dlc_developer_config.toml

```diff
@@ -22,8 +22,8 @@ benchmark_mode = false
 
```

Comment:
In the PR it says `X I've modified dlc_developer_config.toml in my PR branch by setting benchmark_mode = true`. Should the above line be set to true  or is that not relevant here?

Reply:
Unchecked the box. There's no benchmark for AG